### PR TITLE
HL-421: reduce review environment build failures by limiting concurrency of jobs

### PR DESCRIPTION
**Pull request by haavikko**
## Description :sparkles:
Juho Saarinen commented in Slack: "Basically lock should be namespace + service"

NOTE: If this seems to work, I'll add the same config for other YJDH subprojects

### CODE REVIEW

File: .github/workflows/bf-review.yml [hunk 1]
```diff
@@ -55,6 +55,9 @@ jobs:
             context: ./frontend
             dockerfile: ./frontend/benefit/applicant/Dockerfile
             port: 3000
+    concurrency:
+      group: ${{ env.K8S_NAMESPACE }}-${{ matrix.service }}
+      cancel-in-progress: false
```
Comment (Hi-Fi) [inline .github/workflows/bf-review.yml:60]: I think there's no need to block build concurrency, as each build created different image anyway.

File: .github/workflows/bf-review.yml [hunk 2]
```diff
@@ -87,6 +90,9 @@ jobs:
             dockerfile: ./frontend/benefit/applicant/Dockerfile
             database: false
             port: 3000
+    concurrency:
+      group: ${{ env.K8S_NAMESPACE }}-${{ matrix.service }}
```
Comment (Hi-Fi) [inline .github/workflows/bf-review.yml:94]: Namespace is not available here, but only during the execution from `andersinno/kolga-setup-action@v2`.
```suggestion
      group: ${{ github.event.pull_request.number }}-${{ matrix.service }}
```

Comment (haavikko) [reply .github/workflows/bf-review.yml:94]: OK thanks!

**Comment by haavikko** [comment]
After the change suggested by @Hi-Fi , the build got further, but stopped with "timed out waiting for the condition". Retrying. Could be other builds that don't have the concurrency config are causing the error.

**Comment by Hi-Fi** [comment]
> After the change suggested by @Hi-Fi , the build got further, but stopped with "timed out waiting for the condition". Retrying. Could be other builds that don't have the concurrency config are causing the error.

Other namespaces shouldn't directly matter, but without events it's hard to say if more time would help, or where the issue is

**Comment by haavikko** [comment]
Still getting the "timed out waiting for condition" error. Don't know if it's due the new concurrency config or something else.

**Comment by Hi-Fi** [comment]
> Still getting the "timed out waiting for condition" error. Don't know if it's due the new concurrency config or something else.

This fix shouldn't matter to that, but prevent the stage where Helm is stuck on e.g. `pending-upgrade` stage.

Timeout issue can be seen now from events:
```
0/9 nodes are available: 1 node(s) had taint {node.kubernetes.io/disk-pressure: }, that the pod didn't tolerate, 4 Insufficient memory, 6 Insufficient cpu.  
``` 

That would be fixed eventually by putting requests in the level application really needs. Currently (when idleing) apps require only about 1m CPU, and now those reserve 50m per pod.

**Comment by haavikko** [comment]
@Hi-Fi The builds seem to pass now, I did the same changes to all subprojects, if you think this is OK I'll merge it and we'll see if helps with Helm stuckage.

**Review by Hi-Fi — APPROVED** [review]

**Pull request merged**